### PR TITLE
feat(hosting.general): replace privateSQL orders by cloudDB

### PR DESCRIPTION
**Pull request by lucas-chaigne**
ref: WEB-13645

Signed-off-by: Lucas Chaigne <lucas.chaigne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | WEB-13645
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

The goal of this PR is to redirect customer to standard cloudDB when privateSQL is ordered from web application (in hosting/general_information and hosting/databases)

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/database/DATABASE.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
                     <li>
                         <a
                             data-translate="hosting_tab_DATABASES_configuration_order_title_button_private"
-                            data-ui-sref="app.hosting.dashboard.database.order-private"
+                            href="#/order-cloud-db"
```
Comment (frenautvh) [inline packages/manager/apps/web/client/app/hosting/database/DATABASE.html:70]: it's a better practice in angular js to use the ui-sref directive to generate the link with your associated ui router state, in your case is it something like ui-sref="app.private-database-order-clouddb" ?

**Review by frenautvh — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/database/DATABASE.html (hunk 1 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/database/DATABASE.html]: Hello, thanks for the review. I updated the link

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 2]
```diff
@@ -520,9 +520,7 @@
                                 ></span>
                             </oui-action-menu-item>
 
-                            <oui-action-menu-item
-                                data-ui-sref="app.hosting.dashboard.database.order-private"
-                            >
+                            <oui-action-menu-item href="#/order-cloud-db">
```
Comment (anooparveti) [inline packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:523]:
```suggestion
                            <oui-action-menu-item data-href="#/order-cloud-db">
```

Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html]: thanks for the review. element is modified. I actually remplaced href by data-ui-sref

**Review by anooparveti — APPROVED** [review]

**Pull request merged**